app/views/users/_form.html.erb

```diff
@@ -0,0 +1,57 @@
+<!-- <form action="/users" accept-charset="UTF-8" method="post">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+    <label for="username">Username:</label>
+    <input type="text" name="user[username]" id="username"> <br><br>
+    <label for="email">Email:</label>
+    <input type="email" name="user[email]" id="email"> <br><br>
+    <label for="password">Password:</label>
+    <input type="password" name="user[password]" id="password"> <br><br>
+  <input type="submit" name="submit">
+</form> -->
+
+<!-- <%= form_tag("/users") do %>
+<%= label_tag(:username, "Username") %>
+<%= text_field_tag(:username) %> <br><br>
+<%= label_tag(:email, "Email") %>
+<%= email_field_tag(:email) %> <br><br>
+<%= label_tag(:password, "Password") %>
+<%= password_field_tag(:password) %> <br><br>
+<%= submit_tag("Submit") %>
+<% end %> -->
+
+
+<!-- <%= form_with(model: @user) do |form| %>
+<%= form.label :username %>
+<%= form.text_field :username %> <br><br>
+<%= form.label :email %>
+<%= form.text_field :email, placeholder: "example@example.com" %> <br><br>
+<%= form.label :password %>
+<%= form.text_field :password %> <br><br>
+<%= form.submit  %>
+<% end %> -->
+
+<%= form_with(model: @user) do |u| %>
+
+<div>
+  <%= u.label :username %>
+  <%= u.text_field :username %>
+  <% @user.errors.full_messages_for(:username).each do |message| %>
+    <%= message %>
+  <% end %> <br><br>
+</div>
+<div>
+  <%= u.label :email %>
+  <%= u.text_field :email, placeholder: "example@example.com" %>
+  <% @user.errors.full_messages_for(:email).each do |message| %>
+    <%= message %>
+  <% end %> <br><br>
+</div>
```

Comment:
- [ ] Kindly this should be an email field, not a text_field. Kindly make sure it's of type Email. This will ensure that proper data is entered into your application. Kindly you can learn more about it [here](https://guides.rubyonrails.org/form_helpers.html)